tests/test_lab.py

```diff
@@ -58,29 +58,44 @@ def test_ilab_cli_imports():
     subprocess.check_call([sys.executable, "-c", "; ".join(code)], text=True)
 
 
-@pytest.mark.parametrize(
-    "first,second",
-    [
-        # split first and second level for nicer pytest output
-        (None, None),
-        ("config", None),
-        ("config", "init"),
-        ("config", "show"),
-        ("model", None),
-        ("model", "chat"),
-        ("model", "convert"),
-        ("model", "download"),
-        ("model", "evaluate"),
-        ("model", "serve"),
-        ("model", "test"),
-        ("model", "train"),
-        ("data", None),
-        ("data", "generate"),
-        ("system", "sysinfo"),
-        ("taxonomy", None),
-        ("taxonomy", "diff"),
-    ],
-)
+subcommands = [
+    # split first and second level for nicer pytest output
+    # first, second, extra args
+    (None, None, ()),
+    ("config", None, ()),
+    ("config", "init", ()),
+    ("config", "show", ()),
+    ("model", None, ()),
+    ("model", "chat", ()),
+    ("model", "convert", ()),
+    ("model", "download", ()),
+    ("model", "evaluate", ("--benchmark", "mmlu")),
```

Comment:
where is extra args used?

Reply:
It's for https://github.com/instructlab/instructlab/pull/1622 . The PR adds helpers to debug and test CLI parameters and default lookups. I copied the table from the other PR to reduce merge conflicts.

Reply:
```
@pytest.mark.parametrize(
    "first,second,extra",
    # only second level subcommands have a --debug-params option
    [sc for sc in subcommands if sc[1] is not None],
)
```

I see. Ok